spec/spec.md

```diff
@@ -79,32 +79,31 @@ and they are both configured to connect to the same root namespace.
-| GetNamespace     | Describe information of a namespace                                                                                |
-| CreateNamespace  | Create a new namespace under a parent namespace, or root namespace if parent namespace is not specified            |
-| AlterNamespace   | Alter information of a namespace                                                                                   |
-| DropNamespace    | Drop a namespace from its a parent namespace, or root namespace if parent namespace is not specified               |
-| ListTables       | List the names of tables in a namespace                                                                            |
-| TableExists      | Check if a table exists                                                                                            |
-| GetTable         | Describe information of a Lance table in the namespace                                                             |
-| CreateTable      | Create a new Lance table under a namespace                                                                         |
-| RegisterTable    | Register an existing table at a given storage location to a namespace                                              |
-| AlterTable       | Alter information of a Lance table                                                                                 |
-| DropTable        | Drop a table from its namespace                                                                                    |
-| DropTable        | Drop a table from its namespace, also delete its data                                                              |
-| DeregisterTable  | Deregister a table from its namespace, table content is kept unchanged in storage                                  |
-| GetTransaction   | Describe information of a transaction                                                                              |
-| AlterTransaction | Alter information of a transaction                                                                                 |
+| Operation ID      | Description                                                                                                        |
+|-------------------|--------------------------------------------------------------------------------------------------------------------|
+| ListOperations    | List the operations that are supported by this Lance namespace                                                     |
+| ListNamespaces    | List the names of child namespaces in the parent namespace, or root namespace if parent namespace is not specified |
+| NamespaceExists   | Check if a namespace exists                                                                                        |
+| DescribeNamespace | Describe information of a namespace                                                                                |
```

Comment:
I also renamed all the `GetXXX` to `DescribeXXX`, otherwise a route like `POST /table/{id}/get` would feel very weird